swap/protocol.go

```diff
@@ -34,6 +34,9 @@ var (
 	// ErrEmptyAddressInSignature is used when the empty address is used for the chequebook in the handshake
 	ErrEmptyAddressInSignature = errors.New("empty address in handshake")
 
+	// ErrDifferentChainId is used when the chain id exchanged during the handshake does not match
+	ErrDifferentChainId = errors.New("different chain id")
```

Comment:
`var ErrDifferentChainId should be ErrDifferentChainID` -- go-lint

Reply:
I noticed that in the linter output when developing. Looks like I forgot to actually commit that. Interesting that the CI didn't catch this. Should be fixed now.